### DISCUSSION TITLE
如何修改节点ports的属性

### DISCUSSION THREAD

**Discussion by ida-fe** [antvis/XFlow#351] [Q&A]
1、添加节点，并在节点上增加ports
<img width="718" alt="image" src="https://user-images.githubusercontent.com/82094079/217229972-56842729-1046-4bc5-b716-d713b123ae2d.png">
2、发现渲染后ports半径为10，并且没有渲染到对应的位置
<img width="224" alt="image" src="https://user-images.githubusercontent.com/82094079/217230413-370a4232-976c-406e-9221-ff830fe86494.png">

预期下面的效果，哪里设置有问题呢？
<img width="227" alt="image" src="https://user-images.githubusercontent.com/82094079/217230538-fce546b8-2170-4966-bad3-9a0b5d47bf74.png">

**Comment by overtiger** [comment]
我也遇到同样的问题，请问解决了吗
